packages/fxa-settings/src/lib/channels/firefox.ts

```diff
@@ -98,20 +98,8 @@ export type FxALoginRequest = {
 };
 
 // ref: [FxAccounts.sys.mjs](https://searchfox.org/mozilla-central/rev/82828dba9e290914eddd294a0871533875b3a0b5/services/fxaccounts/FxAccounts.sys.mjs#910)
-export type FxALoginSignedInUserRequest = {
+export type FxALoginSignedInUserRequest = FxALoginRequest & {
```

Comment:
I think we probably need a follow up issue because I noticed that we should probably be using this to send the clients `authAt`, reset PW is the only place I see us using it. I didn't check what we do with this on the sync side though.

Reply:
I checked 2/3 clients and we don't use authAt anywhere. 😅  

* Desktop/Android: https://searchfox.org/mozilla-central/search?q=authAt%7Cauth_at&path=services&case=false&regexp=true
* Rust components for Android/iOS: https://github.com/search?q=auth_at+OR+authAt+repo%3Amozilla%2Fapplication-services+&type=code

Reply:
Thanks, I've added a note on FXA-10863!